spec/menu_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'menu'
+
+describe Menu do
+  let(:menu) { Menu.new }
```

Comment:
I think rather than this you could also use 'subject' instead of 'menu' and remove this line.

Up to you what you prefer :)